docs/frontmatter.md

```diff
@@ -88,6 +88,9 @@ The following table lists the available frontmatter fields, a brief description
 * - `subtitle`
   - a string (max 500 chars)
   - page only
+* - `alternatives`
+  - alternative URLs and descriptions for the page content (see [](#alternatives))
+  - page only
```

Comment:
Should this be page only? If we have a zenodo repository that is an alternate, that is for the project.

Reply:
Cool, yeah this is an easy change. Should the project values read through to the page though? i.e. should the zenodo repository be an alternative for every single page that doesn't have a different alternative defined?

Reply:
Maybe not? Certainly not in JATS. They seem like different things, one for the project and one for the page.

Reply:
@fwkoch should we actually go the other way and only allow this on projects to start with? That seems a bit easier to handle. Right now the links from projects get lost when converting to JATS.